app/app.go

```diff
@@ -152,6 +153,11 @@ func New() *cli.App {
 				Value:   10,
 				Usage:   "The number of download thread (only works for multiple-parts video)",
 			},
+			&cli.BoolFlag{
+				Name:    "rename",
+				Aliases: []string{"re"},
+				Usage:   "Avoid duplicate names causing download failures; (name-timestamp)",
```

Comment:
This is `usage`, not reason why we add this flag:

```suggestion
				Usage:   "Rename the file with timestamp suffix",
```